cypher/cypher-docs/src/docs/dev/ql/administration/indexes-for-full-text-search/create-fulltext-syntax.asciidoc

```diff
@@ -6,7 +6,7 @@
 | [source, cypher, role=noplay]
 ----
 CREATE FULLTEXT INDEX [index_name] [IF NOT EXISTS]
-FOR (n:LabelName[\|...])
+FOR (n:LabelName["\|" ...])
```

Comment:
Why is it `index_name` here but just `name` above? Also, why only one pipe here when two pipes in the example above?